src/ci/docker/host-x86_64/dist-various-2/Dockerfile

```diff
@@ -89,14 +89,14 @@ RUN sh /scripts/sccache.sh
 
 ENV CARGO_TARGET_X86_64_FUCHSIA_AR /usr/local/bin/llvm-ar
 ENV CARGO_TARGET_X86_64_FUCHSIA_RUSTFLAGS \
--C link-arg=--sysroot=/usr/local/x86_64-fuchsia \
--C link-arg=-L/usr/local/x86_64-fuchsia/lib \
--C link-arg=-L/usr/local/lib/x86_64-fuchsia/lib
+-C link-arg=--sysroot=/usr/local/x86_64-fuchsia/sysroot \
```

Comment:
(Pre-existing) I think this is redundant with the line below, except when building C/C++ code, which RUSTFLAGS is not used for.

Reply:
Which part is redundant? I was following prior art at https://fuchsia.googlesource.com/infra/recipes/+/d36a3b608399d38367aead9ae9ede46f65101e61/recipes/contrib/rust_toolchain.resources/generate_config.py#167 as the recipe

Reply:
Turns out it's not actually redundant!